cedar-policy-core/Cargo.toml

```diff
@@ -24,6 +24,8 @@ thiserror = "1.0"
 smol_str = { version = "0.2", features = ["serde"] }
 stacker = "0.1.15"
 arbitrary = { version = "1", features = ["derive"], optional = true }
+miette = "5.9.0"
+phf = { version = "0.11.2", features = ["macros"] }
```

Comment:
How essential is this dep?

Reply:
Not very essential. It just makes the parse-token-to-friendly-name mapping faster for error message generation, but the difference vs a regular `match` probably isn't noticeable.